ads/fluct.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {validateData, writeScript} from '../3p/3p';
+
+/* global adingoFluct: false */
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function fluct(global, data) {
+  validateData(data, ['g', 'u']);
+  writeScript(global,
+    `https://cdn-fluct.sh.adingo.jp/f.js?G=${encodeURIComponent(data['g'])}`,
```

Comment:
extra indentation needed for code can't fall in one line. (need 2 more here and below).